.env

```
@@ -1 +1,3 @@
-GETH_HOST_DATA_DIR=./geth-data
\ No newline at end of file
+CLIENT=${CLIENT:-reth}
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The CLIENT key is not assigned properly

(SubstitutionKey)